src/control/dmg/main.go

```diff
@@ -93,6 +117,7 @@ func dmgMain() error {
 
 	_, err := p.Parse()
 	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
Was this debugging code left in by mistake?

Reply:
I initially had this as debug code but since daos_shell can silently fail its parsing and kill itself I think we might want to leave this in to give a heads up of what's going on.